package.json

```diff
@@ -36,6 +36,7 @@
     "sinon": "1.17.6"
   },
   "peerDependencies": {
+    "flow-bin": ">= 0.30.0 < 0.40.0",
```

Comment:
@kumar303 I preferred the explicit version because, at least if I got it correctly from the npmjs docs:

- `^0.2.3 := >=0.2.3 <0.3.0` 

means that `^0.30.0 := >= 0.30.0 <0.31.0`, which sounds a bit too narrow.

It worth to mention that, when the version installed by the main package (e.g. web-ext) is out of the range in the peerDependencies specified here, the only things that happens is that a warning is logged in the console (but this module continues to work as expected, at least if the flow binary has not changed its command line interface in a new version).